vispy/visuals/filters/tests/test_wireframe_filter.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
New files (including the `filters/tests/__init__.py` should copy the license header and copyright.